src/Requests/OrderRequest.php

```diff
@@ -120,7 +120,7 @@ public function addUpdateInvoice(array $parameters = []): Int32Logic4Response
     {
         return Int32Logic4Response::make(
             $this->buildResponse(
-                $this->getClient()->post('/v1.1/Orders/AddUpdateInvoice', ['json' => $parameters]),
+                $this->getClient()->post('/v1.3/Orders/AddUpdateInvoice', ['json' => $parameters]),
```

Comment:
This is a breaking change, as the new endpoint does not support all 'Total' fields anymore.

My suggestion is to upgrade this package to version 3.0, which processes al API versions. To prevent conflicts, all requests will be placed into the correct version namespace (i.e. `V1_2/OrderRequest`). The `latest` version will be discarted.